lib/datapacksutils.js

```diff
@@ -1454,7 +1462,11 @@ DataPacksUtils.prototype.getTitleFieldsForDataPack = async function(sourceDataPa
     var targetPrimarySObject;
     
     if (targetDataPack) {
-        targetPrimarySObject = targetDataPack.VlocityDataPackData[dataField][0];
```

Comment:
Would be better to just call this.getDataField(targetDataPack); again for targetDataPack